lib/iris/tests/unit/time/test_PartialDateTime.py

```diff
@@ -224,12 +224,12 @@ def setUp(self):
         self.expected_value = EQ_EXPECTATIONS
 
     def test_cftime_equal(self):
-        pdt = PartialDateTime(month=3, microsecond=2)
+        pdt = PartialDateTime(month=3, second=2)
```

Comment:
This was failing because the equality operator returned `False`. The microsecond comparison is skipped if `other` doesn't have a microsecond attribute:
https://github.com/SciTools/iris/blob/a6cd220f64f7d2fd2af8308f814d81b00eb1e05e/lib/iris/time.py#L152-L157

My guess is that, when this test was written, `cftime.datetime`s' predecessors did not have a microsecond attribute, but they do now.